jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -792,3 +799,36 @@ function configure_kafka(){
     JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.kie.kafka.server.ext.disabled=true"
   fi
 }
+
+function configure_kafka_jbpm_emitter(){
+  if [ "${KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_ENABLED^^}" = "TRUE" ]; then
+      if [ -n "${KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_BOOTSTRAP_SERVERS}" ];then
+          log_info "Kafka JBPM Emitter enabled"
+          JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.kie.jbpm.event.emitters.kafka.bootstrap.servers=${KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_BOOTSTRAP_SERVERS}"
+          if [ -n "${KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_ACKS}" ];then
+            JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.kie.jbpm.event.emitters.kafka.acks=${KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_ACKS}"
+          fi
+          if [ -n "${KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_CLIENT_ID}" ];then
+            JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.kie.jbpm.event.emitters.kafka.client.id=${KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_CLIENT_ID}"
+          fi
+
+          JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.kie.jbpm.event.emitters.kafka.max.block.ms=${KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_MAX_BLOCK_MS:-2000}"
+
+          if [ -n "${KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_DATE_FORMAT}" ];then
+            JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.kie.jbpm.event.emitters.kafka.date_format=${KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_DATE_FORMAT}"
+          fi
+
+          JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.kie.jbpm.event.emitters.kafka.topic.processes=${KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_PROCESSES_TOPIC_NAME:-"jbpm-processes-events"}"
+          JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.kie.jbpm.event.emitters.kafka.topic.task=${KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_TASK_TOPIC_NAME:-"jbpm-task-events"}"
+          JBOSS_KIE_ARGS="${JBOSS_KIE_ARGS} -Dorg.kie.jbpm.event.emitters.kafka.topic.cases=${KIE_SERVER_KAFKA_JBPM_EVENT_EMITTER_CASES_TOPIC_NAME:-"jbpm-cases-events"}"
+
+          if [ -d "/opt/kie/dependencies/jbpm-kafka" ] ;then
+            mv -v /opt/kie/dependencies/jbpm-clustering/jbpm-event-emitters-kafka-*.jar ${JBOSS_HOME}/standalone/deployments/ROOT.war/WEB-INF/lib
+          fi
+
+      else
+        log_warning "JBPM Emitter Bootstrap servers not configured"
+      fi
```

Comment:
Maybe this would be better to set up as an error. Because without the emitter bootstrap server, Kie Server will start looping logs with not possible to get bootstrap on localhost (used by default by emitter jar)

Reply:
ok

Reply:
I am not sure, the startup problem will happen only if the jar is on the classpath, it is copied only when the minimal requirements are satisfied.

Reply:
yes, you are right @spolti this won't happen as the jar is copied only when the bootstrap env. variable is provided to the image. My bad, I've overlooked it.